_data/projects/url_shortener.yml

```diff
@@ -0,0 +1,11 @@
+name: Urls.ml
+desc: Small set of PHP scripts that will help you in shortening your url by which you can get a more precised version of your url at ease.
+site: https://github.com/urls/url-shortener
+tags:
+- php
+- web
+- mysqli
+- css
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/urls/url-shortener/issues
```

Comment:
It'd be great if this URL pointed to a list of curated issues (if you don't have a dedicated label for these, I'd suggest the `up-for-grabs` label) instead of the full list of issues.